mayastor/src/subsys/opts.rs

```diff
@@ -482,15 +482,36 @@ impl GetOpts for IscsiTgtOpts {
 pub struct ErrStoreOpts {
     /// ring buffer size
     pub err_store_size: usize,
+
     /// NexusErrStore enabled
     pub enable_err_store: bool,
+
+    /// whether to fault the child due to failed re-tried IOs
+    pub fault_child_on_retry_count: bool,
+
+    /// the maximum number of errors for the same retried IO
+    pub max_retry_errors: u32,
```

Comment:
max_retries?

Reply:
This option is now not needed and removed.
But in the same vein renamed the other option max_total_errors to max_errors.